config/environments/development.rb

```diff
@@ -78,23 +85,18 @@
   # Raise error when a before_action's only/except options reference missing actions.
   config.action_controller.raise_on_missing_callback_actions = true
 
+  # Apply autocorrection by RuboCop to files generated by `bin/rails generate`.
+  # config.generators.apply_rubocop_autocorrect_after_generate!
```

Comment:
I like this new option, we could enable it now or later

Reply:
Yeah, I wondered about that too. Decided to leave it commented out since we're going to need to run Prettier on any files too so it doesn't safe us any steps.